lib/ansible/modules/windows/win_command.py

```diff
@@ -60,17 +60,16 @@
 '''
 
 EXAMPLES = r'''
-# Example from Ansible Playbooks.
-- win_command: whoami
+- name: Save the result of 'whoami' in 'whoami_out'
+  win_command: whoami
   register: whoami_out
 
-# Run the command only if the specified file does not exist.
-- win_command: wbadmin -backupTarget:C:\backup\ creates=C:\backup\
+- name: Create a backup only if the file does not exist
+  win_command: wbadmin -backupTarget:C:\backup\
+    creates: C:\backup\
```

Comment:
You know you have a real problem when CI fails and people are discussing why that might be :-P